src/main/java/oshi/util/PdhUtil.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Oshi (https://github.com/dblock/oshi)
+ * 
+ * Copyright (c) 2010 - 2016 The Oshi Project Team
+ * 
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors:
+ * dblock[at]dblock[dot]org
+ * alessandro[at]perucchi[dot]org
+ * widdis[at]gmail[dot]com
+ * https://github.com/dblock/oshi/graphs/contributors
+ */
+package oshi.util;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sun.jna.ptr.IntByReference;
+import com.sun.jna.ptr.PointerByReference;
+
+import oshi.jna.platform.windows.Pdh;
+
+/**
+ * Provides access to open pdh queries and add counters
+ * 
+ * @author widdis[at]gmail[dot]com
+ */
+public class PdhUtil {
```

Comment:
This is an older version than the currently committed PdhUtil.  The newer one is in a platform-specific subdirectory.   You can probably just delete this version of the file from your local branch to make it go away from this PR.